linode/granola-prod/config.nix

```diff
@@ -19,16 +19,10 @@ let
   secretsImportScript = pkgs.writeScript "secrets-import" ''
     #! /bin/sh
     set -eu
-    NETLIFY_AUTH_TOKEN="$(cat /run/keys/netlify-auth-token)"
```

Comment:
Note that the 'prod' build server also builds and deploys other Granola repos besides 'mina-indexer'. They require this Netlify token.